packages/driver/src/cy/commands/navigation.ts

```diff
@@ -673,6 +679,8 @@ export default (Commands, Cypress, cy, state, config) => {
               onFail: options._log,
               args: { str },
             })
+
+            return // Error is thrown above, it exists to satisfy typescript
```

Comment:
Would it make this cleaner to just return what is on line 678?

Reply:
The return type of `throwErrByPath` is `void`, and 678 always throws. So, line 683 is never called. If we remove this line, TypeScript shows `all path doesn't return value` error. That's why I left this comment.

https://github.com/microsoft/TypeScript/issues/13219

In other languages like Java, there is `throws` keyword that handles this situation, but TypeScript isn't. The link above is the request to that feature. It seems that it's forgotten or given low priority.

Reply:
Looking at this comment, if  you update error_utils.throwsErrorByPath to return type never, this should be fine without the extra return type.

https://github.com/Microsoft/TypeScript/issues/13625#issuecomment-274566197